proto/frequenz/api/microgrid/grid.proto

```diff
@@ -0,0 +1,19 @@
+// Contains definitions specific to grid connection points.
+//
+// Copyright:
+// Copyright 2023 Frequenz Energy-as-a-Service GmbH
+//
+// License:
+// MIT
+
+syntax = "proto3";
+
+package frequenz.api.microgrid.grid;
+
+// The grid connection point metadata.
+message Metadata {
+  // The maximum current magnitude in amperes.
+  // This is the maximum current that can be drawn from or supplied to
+  // the grid.
+  uint32 max_current_magnitude = 1;
```

Comment:
Magnitude? I've never experienced us using this term so far. Why couldn't it just be name `max_current` instead?

Reply:
If we wanna introduce this term here we need to describe it better in the comment what that is.

Reply:
The maximum current at a fuse is a signed value. Meaning, if the fuse limit is 5A, then a current _i_ A should be `-5 <= i <= 5`, or `|i| <= 5`. Hence I added the term `magnitude`, to indicate that this limit applies to +veas well as =ve currents.

Or do we want to split this? Then we could specify that at a site we can consume from the grid, but never supply to it.

Reply:
>Or do we want to split this? Then we could specify that at a site we can consume from the grid, but never supply to it.

I made the changes.

Reply:
> I made the changes

Don't! This is the fuse limit. A fuse imho doesn't care in which direction the power is flowing. This I would call this max_grid_fuse_current or something tbh. I would suggest to later introduce virtual limits for grid feed-in or grid consumption once we need that. More likely we will add frequency limits later on as the first step but lets postpone that discussion until we really need it.

Reply:
Done. I renamed the variable to `max_grid_fuse_current`.

Reply:
is this checked in?  I still see 3 vars, one for each phase.

Reply:
Not yet. I am waiting for further input from Niklas. I will update the PR once I have all the details.

Reply:
Renamed it to `rated_fuse_current`.

https://github.com/frequenz-floss/frequenz-api-microgrid/pull/38#issuecomment-1514732619